lib/test/t_IntegrationExpansion_std.cxx

```diff
@@ -0,0 +1,154 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The test file of the IntegrationExpansion class
+ *
+ *  Copyright 2005-2022 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/OT.hxx"
+#include "openturns/OTtestcode.hxx"
+
+using namespace OT;
+using namespace OT::Test;
+
+int main(int, char *[])
+{
+  TESTPREAMBLE;
+  OStream fullprint(std::cout);
+  setRandomGenerator();
+
+  try
+    {
```

Comment:
The code seems to be wrongly indented, according to the current practices in the C++ code, see e.g. [LeastSquaresStrategy.cxx#L125](https://github.com/openturns/openturns/blob/0e35e69aa44f76d7f8af4b25b8cad21f2f1dadac/lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/LeastSquaresStrategy.cxx#L125)
```suggestion
  {
```

Reply:
It is due to the way emacs indents code by default. If I use the indent.sh script in the utils folder I get these files modified:
```
 M lib/src/Base/Common/Catalog.cxx
 M lib/src/Base/Common/Log.cxx
 M lib/src/Base/Common/ResourceMap.cxx
 M lib/src/Base/Common/TBBImplementation.cxx
 M lib/src/Base/Common/XMLStorageManager.cxx
 M lib/src/Base/Common/openturns/MutexLock.hxx
 M lib/src/Base/Func/SpecFunc/SpecFunc.cxx
 M lib/src/Base/Geom/IntervalMesher.cxx
 M lib/src/Base/Geom/LevelSet.cxx
 M lib/src/Base/Optim/Bonmin.cxx
 M lib/src/Base/Optim/BonminProblem.cxx
 M lib/src/Base/Optim/Dlib.cxx
 M lib/src/Base/Optim/NLopt.cxx
 M lib/src/Base/Optim/OptimizationResult.cxx
 M lib/src/Base/Optim/Pagmo.cxx
 M lib/src/Base/Type/MatrixImplementation.cxx
 M lib/src/Base/Type/openturns/IndicesCollectionImplementation.hxx
 M lib/src/Base/Type/openturns/MatrixImplementation.hxx
 M lib/src/Uncertainty/Algorithm/Classification/MixtureClassifier.cxx
 M lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosSobolIndices.cxx
 M lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GeneralLinearModelAlgorithm.cxx
 M lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingEvaluation.cxx
 M lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingResult.cxx
 M lib/src/Uncertainty/Algorithm/Sensitivity/HSICEstimatorImplementation.cxx
 M lib/src/Uncertainty/Algorithm/Sensitivity/HSICStatImplementation.cxx
 M lib/src/Uncertainty/Algorithm/Sensitivity/HSICUStat.cxx
 M lib/src/Uncertainty/Algorithm/Sensitivity/HSICVStat.cxx
 M lib/src/Uncertainty/Algorithm/Sensitivity/openturns/HSICVStat.hxx
 M lib/src/Uncertainty/Algorithm/Simulation/NAIS.cxx
 M lib/src/Uncertainty/Algorithm/Transformation/ProcessTransformation/BoxCoxFactory.cxx
 M lib/src/Uncertainty/Bayesian/CalibrationResult.cxx
 M lib/src/Uncertainty/Distribution/RandomMixture.cxx
 M lib/test/t_Bonmin_std.cxx
 M lib/test/t_DistFunc_binomial.cxx
 M lib/test/t_FunctionalChaosSobolIndices_gsobol.cxx
 M lib/test/t_FunctionalChaosSobolIndices_ishigami.cxx
 M lib/test/t_HSICStat_std.cxx
 M lib/test/t_Indices_std.cxx
 M lib/test/t_IntegrationExpansion_std.cxx
 M lib/test/t_KernelSmoothing_std.cxx
 M lib/test/t_Log_std.cxx
 M python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py
 M python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_flooding.py
 M python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_generate_chaboche.py
 M python/doc/examples/data_analysis/manage_data_and_samples/plot_quick_start_point_and_sample.py
 M python/doc/examples/data_analysis/sample_analysis/plot_compare_unconditional_conditional_histograms.py
 M python/doc/examples/meta_modeling/general_purpose_metamodels/plot_stepwise.py
 M python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_chose_trend.py
 M python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_multioutput_firesatellite.py
 M python/doc/examples/meta_modeling/kriging_metamodel/plot_propagate_kriging_ishigami.py
 M python/doc/examples/reliability_sensitivity/design_of_experiments/plot_smolyak_merge.py
 M python/doc/examples/reliability_sensitivity/reliability/plot_form_explained.py
 M python/doc/examples/reliability_sensitivity/reliability/plot_nais.py
 M python/doc/examples/reliability_sensitivity/reliability/plot_subset_sampling.py
 M python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_wingweight.py
 M python/src/coupling_tools.py
 M python/src/viewer.py
 M python/test/t_Basis_std.py
 M python/test/t_Bernoulli_std.py
 M python/test/t_BernsteinCopulaFactory_std.py
 M python/test/t_DistFunc_binomial.py
 M python/test/t_Distribution_std.py
 M python/test/t_Event_system.py
 M python/test/t_FunctionalChaos_mixed.py
 M python/test/t_GaussianNonLinearCalibration_std.py
 M python/test/t_HSICEstimatorConditionalSensitivity_std.py
 M python/test/t_HSICEstimatorGlobalSensitivity_std.py
 M python/test/t_HSICEstimatorTargetSensitivity_std.py
 M python/test/t_HSICStat_std.py
 M python/test/t_HypothesisTest_std.py
 M python/test/t_Indices_std.py
 M python/test/t_IntegrationExpansion_std.py
 M python/test/t_InverseGamma_std.py
 M python/test/t_KernelSmoothing_std.py
 M python/test/t_KrigingAlgorithm_std.py
 M python/test/t_LeastSquaresExpansion_std.py
 M python/test/t_MemoizeFunction_std.py
 M python/test/t_NLopt_std.py
 M python/test/t_Pagmo_std.py
 M python/test/t_SpecFunc_std.py
 M python/test/t_Viewer.py
 M python/test/t_Waarts_discontinuous_lsf.py
 M python/test/t_docstring_missing.py
```
As you see, it is better to fix these problems during the final integration step rather than on the fly.